src/lib/slider/slider.scss

```diff
@@ -106,8 +108,8 @@ $mat-slider-focus-ring-size: 30px !default;
     @include cursor-grab;
   }
 
-  .mat-slider:not(.mat-slider-disabled) &:active,
-  .mat-slider-sliding:not(.mat-slider-disabled) & {
+  &.mat-slider:not(.mat-slider-disabled) &:active,
```

Comment:
This seems off. When it's used on `.mat-slider` it'll produce `.mat-slider.mat-slider:not(.mat-slider-disabled) &:active` whereas on `.mat-slider-thumb` and `.mat-slider-thumb-label` it won't match at all.